mitodl/open-discussions
# add mobile display, consistent layout

#### What are the relevant tickets?

this closes #115 and addresses parts of #85 

#### What's this PR do?

- add a toolbar (along the top)
- add a mobile drawer
- set up a consistent, responsive layout for every page
- add a desktop navigation sidebar

thats about it. Here's what it looks like now:

![fjfjfjfjdddddd](https://user-images.githubusercontent.com/6207644/29682662-d180d3b6-88d9-11e7-8517-d960626a0089.png)
![fjfjfjfjdddd](https://user-images.githubusercontent.com/6207644/29682661-d17cfb56-88d9-11e7-8a80-62ea035c819f.png)
![fjfjfjfjddd](https://user-images.githubusercontent.com/6207644/29682660-d17d095c-88d9-11e7-84da-9b359055e64c.png)

#### How should this be manually tested?

There's a few things:

- the 'menu' icon for the drawer should not show up on a desktop width viewport
- everything should be usable and legible on a phone width viewport
- the layout for all pages should look reasonable and be the same
- the navigation sidebar should show up on the frontpage (`/`) and on the channel page
- the 'create post' button should show up on the channel page, and should take you to the right place
